### PR TITLE
here is the finalPage results

### PR DESCRIPTION
**Pull request by alqayadhi**
I hope it works as it should be :) below is a screen shoot of the results 

image

### CODE REVIEW

File: src/pages/questionPage.js [hunk 1]
```diff
@@ -45,7 +46,13 @@ export const initQuestionPage = () => {
 
 const nextQuestion = () => {
   quizData.currentQuestionIndex = quizData.currentQuestionIndex + 1;
-  initQuestionPage();
+  initQuestionPage(); 
+  //
+  if (quizData.currentQuestionIndex === quizData.questions.length - 1) {
```
Comment (arkaitzgarro) [inline src/pages/questionPage.js:51]: The condition is not correct, the quiz ends on the 9th question not in the 10th.

File: src/pages/questionPage.js [hunk 2]
```diff
@@ -45,7 +46,13 @@ export const initQuestionPage = () => {
...
+    //localStorage.clear();
```
Comment (arkaitzgarro) [inline src/pages/questionPage.js:52]: This can be removed.

File: src/views/selectedAnswerView.js [hunk 3]
```diff
@@ -1,6 +1,13 @@
+import {createFinalSummaryElement} from "../views/finalView.js";
+import {quizData} from "../data.js";
+
 export const selectAnswerVariant = (selectedElement, rightAnswer) => {
+  
   if (selectedElement === rightAnswer) {
     selectedElement.classList.add('selected-green');
+    quizData.finalScore++
+    // print the finalScores
+    console.log(`Your answer is = ${quizData.finalScore}`);
```
Comment (arkaitzgarro) [inline src/views/selectedAnswerView.js:10]: This can be removed

File: src/pages/questionPage.js [hunk 4]
```diff
@@ -45,7 +46,13 @@ export const initQuestionPage = () => {
 
 const nextQuestion = () => {
   quizData.currentQuestionIndex = quizData.currentQuestionIndex + 1;
-  initQuestionPage();
+  initQuestionPage(); 
+  //
```
Comment (arkaitzgarro) [inline src/pages/questionPage.js:50]: This can be removed.

**Review by arkaitzgarro — COMMENTED** [review]
One small adjustment and we are good to go.

File: src/pages/questionPage.js (hunk 1 above)
Comment (alqayadhi) [reply src/pages/questionPage.js:51]: yeah i noticed that I will fix it immediately

File: src/pages/questionPage.js (hunk 2 above)
Comment (alqayadhi) [reply src/pages/questionPage.js:52]: copy that

File: src/views/selectedAnswerView.js (hunk 3 above)
Comment (alqayadhi) [reply src/views/selectedAnswerView.js:10]: I forget to delate it, I made to check the console :/

File: src/pages/questionPage.js (hunk 4 above)
Comment (alqayadhi) [reply src/pages/questionPage.js:50]: done

**Review by arkaitzgarro — APPROVED** [review]
@alqayadhi I moved the check to the page file again. Ideally, view components don't have a knowledge of pages, only of other views. That way, the business logic stays in parent components, and the view can be reused.

**Pull request merged**